src/main/resources/scripts/furnace automatisation.sk

```diff
@@ -15,14 +15,15 @@ options:
 	radius: 1
 
 on fuel burn:
-	fuel slot of block will be empty
+	fuel slot of event-block will be empty
+	wait 1 second
 	loop blocks in radius {@radius}:
-		loop-block is chest
-		loop items of type fuel:

```

Comment:
How it this an improvement? Inventories are most commonly larger than amount of fuels that Minecraft has.

Reply:
It didn't work before, so I recoded it this way, and it still doesn't work, but it works more than before, it atleast removes the fuel from the chest now.